core/capabilities/integration_tests/keystone/workflow.go

```diff
@@ -134,3 +134,50 @@ func createLLOStreamWorkflowJob(t *testing.T,
 		consumerAddr.String()))
 	return workflowJobSpec.Job()
 }
+
+const secureMintWorkflow = `
+name: "%s"
+owner: "0x%s"
+triggers:
+  - id: "securemint-trigger@1.0.0"
+    config:
+      maxFrequencyMs: 5000
+
+consensus:
+  - id: "offchain_reporting@1.0.0"
+    ref: "secure-mint-consensus"
+    inputs:
+      observations:
+        - "$(trigger.outputs)"
+    config:
+      report_id: "0003"  
+      key_id: "evm"	
+      aggregation_method: "secure_mint" #NEW AGGREGRATION METHOD
+      aggregation_config:
+        targetChainSelector:
+          "%d" # CHAIN_ID_FOR_WRITE_TARGET: NEW Param, to match write target
+      encoder: "EVM"
+      encoder_config:
+        abi: "(bytes32 FeedID, uint224 Price, uint32 Timestamp)[] Reports"
+
+targets:
+  - id: "write_geth-testnet@1.0.0"
+    inputs:
+      signed_report: $(secure-mint-consensus.outputs)
+    config:
+      address: "%s"
+      params: ["$(report)"]
+      abi: "receive(report bytes)"
+      deltaStage: 1s
+      schedule: oneAtATime
+`
+
+func createSecureMintWorkflowJob(t *testing.T,
+	workflowName string,
+	workflowOwner string,
+	chainSelector int64,
+	consumerAddr common.Address) job.Job {
+	spec := fmt.Sprintf(secureMintWorkflow, workflowName, workflowOwner, chainSelector, consumerAddr.String())
```

Comment:
Explicit `.String()` call isn't necessary:
```suggestion
	spec := fmt.Sprintf(secureMintWorkflow, workflowName, workflowOwner, chainSelector, consumerAddr)
```
On the other hand, this is pretty tough to read being such a large message. Might want to consider swapping in the template package.

Reply:
Done!